docs/setup/production.asciidoc

```diff
@@ -112,6 +112,15 @@ cluster.name: "my_cluster"
 # The Elasticsearch instance to use for all your queries.
 elasticsearch.url: "http://localhost:9200"
 --------
+. Check your transport and HTTP host configs. Kibana wants to connect to localhost:9200, but your client node needs to be reachable to the other members of the cluster on the transport port.
```

Comment:
Can you mention `elasticsearch.yml` specifically here just like we do in previous sections?  It'd probably be good to move this up a bit as well so people aren't jumping from elasticsearch.yml to kibana.yml.